.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Thinking about this, I'm 50/50 on whether or not this is a good idea for this repo; this will prevent people from modifying the file without it being dirty on disk, but I also think that it's not going to end up being used because it's a lot more ergonomic to open a specific types folder instead of the entire repo.

Reply:
I'm always doing `types/*`, never repo, that's a valid point

Reply:
Interesting, I actually always do repo.

Reply:
Changed to a `.vscode/settings.template.json`.